File: gymgenie-mobile/composeApp/src/androidMain/kotlin/com/asc/gymgenie/feature/create_workout/MuscleGroupEmoji.kt
package com.asc.gymgenie.feature.create_workout

import com.asc.gymgenie.R

internal fun muscleGroupDrawable(key: String): Int = when (key.uppercase()) {
    "CHEST" -> R.drawable.ic_chest
    "BACK" -> R.drawable.ic_back
    "SHOULDERS" -> R.drawable.ic_shoulders
    "BICEPS" -> R.drawable.ic_biceps
    "TRICEPS" -> R.drawable.ic_triceps
    "FOREARMS" -> R.drawable.ic_forearms
    "ABS" -> R.drawable.ic_abs
    "QUADRICEPS" -> R.drawable.ic_quadriceps
    "HAMSTRINGS" -> R.drawable.ic_hamstrings
    "GLUTES" -> R.drawable.ic_glutes
    "CALVES" -> R.drawable.ic_calves
    "FULL_BODY" -> R.drawable.ic_fullbody
    "CARDIO" -> R.drawable.ic_cardio
    else -> R.drawable.ic_fullbody
}

internal fun muscleGroupPickerEmoji(key: String): String = when (key.uppercase()) {
    "CHEST" -> "🫁"
    "BACK" -> "🦾"
    "SHOULDERS" -> "💪"
    "BICEPS" -> "💪"
    "TRICEPS" -> "🤸"
    "FOREARMS" -> "✊"
    "ABS" -> "🔥"
    "QUADRICEPS" -> "🦵"
    "HAMSTRINGS" -> "🦵"
    "GLUTES" -> "🍑"
    "CALVES" -> "🦶"
    "FULL_BODY" -> "⭐"
    "CARDIO" -> "❤️"
    else -> "🏋"
}

File: gymgenie-mobile/iosApp/iosApp/CreateWorkout/MuscleGroupEmoji.swift
import Foundation

func muscleGroupImageName(_ key: String) -> String {
    switch key.uppercased() {
    case "CHEST":       return "ic_chest"
    case "BACK":        return "ic_back"
    case "SHOULDERS":   return "ic_shoulders"
    case "BICEPS":      return "ic_biceps"
    case "TRICEPS":     return "ic_triceps"
    case "FOREARMS":    return "ic_forearms"
    case "ABS":         return "ic_abs"
    case "QUADRICEPS":  return "ic_quadriceps"
    case "HAMSTRINGS":  return "ic_hamstrings"
    case "GLUTES":      return "ic_glutes"
    case "CALVES":      return "ic_calves"
    case "FULL_BODY":   return "ic_fullbody"
    case "CARDIO":      return "ic_cardio"
    default:            return "ic_fullbody"
    }
}

func muscleGroupEmoji(_ key: String) -> String {
    switch key.uppercased() {
    case "CHEST":       return "🫁"
    case "BACK":        return "🦾"
    case "SHOULDERS":   return "💪"
    case "BICEPS":      return "💪"
    case "TRICEPS":     return "🤸"
    case "FOREARMS":    return "✊"
    case "ABS":         return "🔥"
    case "QUADRICEPS":  return "🦵"
    case "HAMSTRINGS":  return "🦵"
    case "GLUTES":      return "🍑"
    case "CALVES":      return "🦶"
    case "FULL_BODY":   return "⭐"
    case "CARDIO":      return "❤️"
    default:            return "💪"
    }
}
